test/single/test_run.py

```diff
@@ -746,16 +748,23 @@ def test_mpi_run_minimal(self):
         settings = self.minimal_settings
 
         def mpi_impl_flags(tcp, env=None):
-            return ["--mock-mpi-impl-flags"], ["--mock-mpi-binding-args"]
+            return ["--mock-mpi-impl-flags"], ["--mock-mpi-binding-args"], None
 
         with mock.patch("horovod.runner.mpi_run._get_mpi_implementation_flags", side_effect=mpi_impl_flags):
             with mock.patch("horovod.runner.mpi_run.safe_shell_exec.execute", return_value=0) as execute:
                 mpi_run(settings, None, {}, cmd)
 
                 # call the mocked _get_mpi_implementation_flags method
-                mpi_flags, binding_args = horovod.runner.mpi_run._get_mpi_implementation_flags(False)
+                mpi_flags, binding_args, mpi = horovod.runner.mpi_run._get_mpi_implementation_flags(False)
                 self.assertIsNotNone(mpi_flags)
-                expected_cmd = ('mpirun '
+                if _IMPI_IMPL == mpi:
+                    expected_cmd = ('mpirun '
+                                '-l -n 2 -hosts localhost '
```

Comment:
indentation looks funny here and in else branch

Reply:
Fixed